pkgs/top-level/python-packages.nix

```diff
@@ -759,6 +759,8 @@ in {
 
   fuzzywuzzy = callPackage ../development/python-modules/fuzzywuzzy { };
 
+
+
```

Comment:
why move? it was in a better ordered position orginially

Reply:
Removed these 2 blank lines